subsys/canbus/isotp/isotp.c

```diff
@@ -432,6 +432,7 @@ static inline void receive_add_mem(struct isotp_recv_ctx *ctx, u8_t *data,
 	if (!ctx->act_frag) {
```

Comment:
In commit message "canbus: isotp: Fix NULL deref after overflow", better would be to have it like this, to make these one look like in other PRs.
```
Fixes #22657
Coverity-CID: 208191
```

Reply:
Done